File: code-coaches/medium/symbols/Symbols.java
import java.util.Scanner;

public class Symbols {

	public static String removeSymbols(String encoded) {
	    // symbols: a char that is not alphanumeric or a space
	    String decoded = "";

		for (char c : encoded.toCharArray()) {
			if (Character.isLetterOrDigit(c) || Character.isWhitespace(c)) {
				decoded += c;
			}
		}

		return decoded;
	}

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String encoded = input.nextLine();

		System.out.println(removeSymbols(encoded));

		input.close();
	}

}

File: code-coaches/medium/symbols/Symbols.cs
using System;

public class Symbols
{
	static string RemoveSymbols(string encoded)
	{
	    // symbols: a char that is not alphanumeric or a space
	    string decoded = "";

		foreach (char c in encoded.ToCharArray())
		{
			if (char.IsLetterOrDigit(c) || char.IsWhiteSpace(c))
				decoded += c;
		}

		return decoded;
	}

	static void Main(string[] args)
	{
		string encoded = Console.ReadLine();
		Console.WriteLine(RemoveSymbols(encoded));
	}
}
